wandb/sdk/wandb_settings.py

```diff
@@ -340,8 +357,30 @@ def _probe(self):
             if self.jupyter:
                 console = "off"
             u["console"] = console
+        u["save_code"] = env.should_save_code()
+        u["disable_code"] = os.getenv(env.DISABLE_CODE)
         self.update(u)
 
+        self._load_viewer_settings()
+
+    def _load_viewer_settings(self):
```

Comment:
I know @raubitsj had some initial scaffolding for loading server settings.  @raubitsj should we wire that in now?

Reply:
yes, I will do it in a separate PR after this gets in.